local_deploy/encode_query.py

```diff
@@ -0,0 +1,60 @@
+import base64
+from io import BytesIO
+import os
+import time
+import numpy as np
+from square_auth.client_credentials import ClientCredentials
+import requests
+
+
+def get_token() -> str:
+    client_credentials = ClientCredentials(
+        keycloak_base_url="",
+        buffer=60,
+    )
+    return client_credentials()
+
+
+def _decode_embeddings(encoded_string: str):
+    encoded_string = encoded_string.encode()
+    arr_binary = base64.decodebytes(encoded_string)
+    arr = np.load(BytesIO(arr_binary))
+    return arr
+
+
+def encode_query() -> None:
+    response = requests.post(
+        # "https://host.docker.internal/api/main/msmarco-distilbert-base-tas-b/embedding",
+        # "http://localhost/api/main/msmarco-distilbert-base-tas-b/embedding",
+        "http://traefik/api/main/msmarco-distilbert-base-tas-b/embedding",
+        json={
+            "input": ["1 in 5 million in UK have abnormal PrP positivity."],
+            "adapter_name": None,
+            "task_kwargs": {"embedding_mode": "cls"},
+        },
+        headers={"Authorization": f"Bearer {get_token()}"},
+        verify=False,
+        allow_redirects=True
+    )
+    print(response)
```

Comment:
do we need this print?

Reply:
[683dd5a](https://github.com/UKP-SQuARE/square-core/pull/354/commits/683dd5af7cfd8116b4e3ac4a9a874ed8d2b31723)